benchmarks/lib/src/jmh/java/com/amazon/corretto/crypto/provider/benchmarks/AesGcmOneShot.java

```diff
@@ -1,8 +1,10 @@
 // Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-package com.amazon.corretto.crypto.provider;
```

Comment:
When using published ACCP for benchmarking, it complains that the benchmark package is not consistent with ACCP's signed JAR. To resolve this issue, the benchmark code is moved to `com.amazon.corretto.crypto.provider.benchmarks` from `com.amazon.corretto.crypto.provider`.